px-context-browser.html

```diff
@@ -247,6 +275,16 @@ <h1 class="epsilon caps weight--normal u-m0" on-click="toggleColumnBrowser">
           value: function() {return {};}
         },
         /**
+         * If we have a selected multiple Items, it's stored here.
+         * must be initially empty for polymer to determine the correct classes for ULs and LIs if there's not selected Asset
+         * @type {Array}
+         * @default ''
+         */
+        selectedItems: {
+          type: Array,
+          value: []
+        },
```

Comment:
Right now, I'm not sure how the developer should expect updates when the `selectedItems` array is updated. If you want to fire property change events/allow data-binding, you should set `notify: true` on the `selectedItems` ([see the Polymer data-flow docs](https://www.polymer-project.org/1.0/docs/devguide/data-system#data-flow)). 

In addition, you may want to fire some custom events with `this.fire` when selection events happen (e.g. when the checkbox is ticked/unticked for an item and it is added/removed from the selected items array).